build/Helix/ProcessHelixFiles.ps1

```diff
@@ -132,7 +139,17 @@ foreach ($testRun in $testRuns.value)
                     }
                     catch
                     {
-                        Log-Error "Failed to download $($file.Name): $($_.Exception.Message)"
+                        Log-Error "Failed to download $($verificationFile.Name) to $destination : $($_.Exception.Message) -- URL: $($verificationFile.Link)"
+                        Write-Host "Trying backup download"
+                        try
+                        {
+                            $backupDestination = "$visualTreeVerificationFolder\$($verificationFile.Name)"
```

Comment:
What is the purpose of this?

Reply:
This was the original destination string, I made this change because it fixed an issue locally, but I now wonder if this was the cause of the new issue since the destination must have been valid before this series of changes.